e2e/playwright/support/utils.ts

```diff
@@ -1,22 +1,36 @@
-import path from 'node:path';
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
 
 import type {Page} from '@playwright/test';
 
 import {ChannelsPage} from '@e2e-support/ui/pages';
 import {UserProfile} from '@mattermost/types/users';
 import Client4 from '@mattermost/client/client4';
 
-const SCREENSHOTS_DIR = path.join(__dirname, '../screenshots');
+export const waitForNewMessages = async (page: Page) => {
+    await page.waitForTimeout(1000);
```

Comment:
the smnart wait for New-messages does not work in CI for some tests.

I'll research more, but I see some value in having green e2e until then.